ldpd/ldpd.c

```diff
@@ -140,6 +140,7 @@ static struct option longopts[] =
 	{ "ctl_socket",  required_argument, NULL, OPTION_CTLSOCK},
 	{ "user",        required_argument, NULL, 'u'},
 	{ "group",       required_argument, NULL, 'g'},
+	{ "instance",    required_argument, NULL, 'n'},
```

Comment:
Well, in our use case we definitely need it. As I said during our last call, we are using the external label manager, so every LDP connects to its zebra pair which just relays label requests to a common external one. So this external label manager needs a way to distinguish between different LDPs requesting labels.